Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: clean deps fmt vet test docker
+
+EXECUTABLE ?= drone-pypi
+IMAGE ?= plugins/$(EXECUTABLE)
+CI_BUILD_NUMBER ?= 0
+
+LDFLAGS = -X "main.buildDate=$(shell date -u '+%Y-%m-%d %H:%M:%S %Z')"
+PACKAGES = $(shell go list ./... | grep -v /vendor/)
+
+clean:
```

Comment:
does the makefile default to the first task as default? If yes I agree `all: build` should be the default task as is standard for most projects. I don't think we need to retroactively update all projects, but we can adjust individually, as needed